tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml

```
@@ -248,7 +243,7 @@ spec:
         - "$(params.REKOR_HOST)"
         - "--ignore-rekor=$(params.IGNORE_REKOR)"
         - "--workers"
-        - "$(params.WORKERS)"
+        - "35"
```

I know the commit message explains it well, but even so:

```suggestion
        # Temporarily hard code this instead of using the WORKERS param
        - "35"
```